server/middleware/coppa-deny.coffee

```diff
@@ -0,0 +1,20 @@
+sendwithus = require '../sendwithus'
+utils = require '../lib/utils'
+errors = require '../commons/errors'
+wrap = require 'co-express'
+database = require '../commons/database'
+parse = require '../commons/parse'
+
+module.exports =
+  sendParentSignupInstructions: wrap (req, res) ->
+    context =
+      email_id: sendwithus.templates.coppa_deny_parent_signup
+      recipient:
+        address: req.body.parentEmail
+    sendwithus.api.send context, (err, result) ->
+      console.log err
+      console.log result
+      if err
+        res.status(400).send { error: err }
```

Comment:
This seems like an internal server error, not a bad request error, which we don't really use anyway. And when possible, use the `errors` module classes. So

```
next(new errors.InternalServerError('Error sending message'))
```

That way the client gets a standard error JSON object, as well.